script.js

```diff
@@ -3,7 +3,7 @@ const burgerMenu = document.querySelector('.nav-menu,.nav-menu-click');
 const burgerImg = document.querySelector('.nav-menu-img');
 const mainContainer = document.querySelector('.main-container');
 const menuIcon = document.querySelector('.icon');
-const menuList = document.querySelectorAll('.top-menu-list li');
+// const menuList = document.querySelectorAll('.top-menu-list li');
```

Comment:
- [ ] Commented codes are supposed to be removed from the codebase because they don't make it look clean. 😃 👍 So please remove them